crates/sui-cost-tables/src/bytecode_tables.rs

```diff
@@ -150,7 +150,6 @@ fn get_simple_instruction_opcode(instr: SimpleInstruction) -> Opcodes {
         BrFalse => BR_FALSE,
         Branch => BRANCH,
 
-        Pop => POP,
```

Comment:
Is this an intentional change?

Reply:
This is part of refactoring to upgrade Move at the same time at these changes, which was ill-advised :/. I will rebase on https://github.com/MystenLabs/sui/commit/fc2e646963105951907555c194d0eba475897050 and hopefully the PR will be much smaller/less confusing :)